test/k6/rollup.config.js

```diff
@@ -0,0 +1,41 @@
+import commonjs from '@rollup/plugin-commonjs'
+import json from '@rollup/plugin-json'
+import resolve from '@rollup/plugin-node-resolve'
+import babel from 'rollup-plugin-babel'
+import { terser } from 'rollup-plugin-terser'
+import multiInput from 'rollup-plugin-multi-input';
+import path from 'path';
+
+const extensions = ['.js', '.jsx', '.ts', '.tsx']
```

Comment:
`jsx` and `tsx` leftover from some react? I think we won't have view files here, orrrr?

Reply:
yea its a left over, it can be removed